azure-webapp-maven-plugin/README.md

```diff
@@ -1,182 +1,175 @@
+    You can find v1 schema [here](v1-schema.md)
+
+    Property | Required | Description | Version
+    ---|---|---|---
+    [region](#region) | true | Specifies the region where your Web App will be hosted; the default value is **westus**. All valid regions at [Supported Regions](#region) section. | 0.1.0+
+    `<resourceGroup>` | true | Azure Resource Group for your Web App. | 0.1.0+
+    `<appName>` | true | The name of your Web App. | 0.1.0+
+    [pricingTier](#pricingtier) | false | The pricing tier for your Web App. The default value is **P1V2**.| 0.1.0+
+    `<deploymentSlot>` | false | The deployment slot to deploy your application. | 1.3.0+
+    `<appServicePlanResourceGroup>` | false | The resource group of the existing App Service Plan. If not specified, the value defined in `<resourceGroup>` will be used by default. | 1.0.0+
+    `<appServicePlanName>` | false | The name of the existing App Service Plan. | 1.0.0+
+    `<appSettings>` | false | Specifies the application settings for your Web App. You could find detail samples for appsettings at [here](../docs/web-app-samples.md). | 0.1.0+
+    `<stopAppDuringDeployment>` | false | To stop the target Web App or not during deployment. This will prevent deployment failure caused by IIS locking files. | 0.1.4+
+
+- Runtime settings
 
     Supported `<os>` values are *Linux*, *Windows* and *Docker*.
     Only the `jre8` is supported for `<javaVersion>` for Web App on Linux.
     If the `<webContainer>` is not configured and the `<os>` is Windows, tomcat 8.5 will be used as default value.
     But if the `<os>` is Linux, the web app will use the JavaSE as runtime.
-    
+
     The runtime settings of v2 configuration could be omitted if users specify an existing web app in the configuration and just want to do the deploy directly.
     ```xml
     <configuration>
-    ...
-    <runtime>
-      <os>Linux</os>
-      <javaVersion>jre8</javaVersion>
-      <webContainer></webContainer>
-    </runtime>
-    <!-- os -->
-    <runtime>
-      <os>Docker</os>
-      <!-- only image is required -->
-      <image>[hub-user/]repo-name[:tag]</image>
-      <serverId></serverId>
-      <registryUrl></registryUrl>
-     </runtime>
+        ...
+        <runtime>
+            <os>Linux</os>
+            <javaVersion>jre8</javaVersion>
+            <webContainer></webContainer>
+        </runtime>
+            <!-- os -->
+        <runtime>
+            <os>Docker</os>
+            <!-- only image is required -->
+            <image>[hub-user/]repo-name[:tag]</image>
+            <serverId></serverId>
+            <registryUrl></registryUrl>
+        </runtime>
     </configuration>
     ```
-
-3. Deployment settings
+- Deployment settings
 
     Users don't need to care about the deployment type in v2 configuration.
     Just configure the resources to deploy to the Web App.
-    
+
     It will use the zip deploy for fast and easy deploy.
```

Comment:
Add a link to zip deploy and war deploy.
- Zip Deploy: https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file
- War Deploy: https://github.com/projectkudu/kudu/wiki/Deploying-WAR-files-using-wardeploy